pyface/action/api.py

```diff
@@ -9,6 +9,68 @@
 # Thanks for using Enthought open source!
 
 
+"""
+
+API for the ``pyface.action`` submodule.
+
+Action and subclasses
+---------------------
+
+- :class:`~.Action`
+- :class:`~.FieldAction`
+- :class:`~.GUIApplicationAction`
+- :class:`~.ListeningAction`
+- :class:`~.TraitsUIWidgetAction`
+- :class:`~.WindowAction`
+
+Action Controller
+-----------------
+
+- :class:`~.ActionController`
+
+Action Event
+------------
+
+- :class:`~.ActionEvent`
+
+Action Manager and subclasses
```

Comment:
Perhaps the mention of "subclasses" can be removed?  This documentation can stay relatively high-level, the MROs are details.  The API documentation for each of these classes will mention what they are subclassing.  Then you would not need to add the note about ``StatusBarManager``. It feels a bit out of place.

Even better would be section titles that hint what these classes are useful for.

Reply:
> This documentation can stay relatively high-level, the MROs are details.

Good point. Removed the "... and subclasses" part in the section title and the separation between toolkit-specific implementations vs the base class and the comment about `StatusBarManager`.

> Even better would be section titles that hint what these classes are useful for.

Not entirely sure how best to phrase that other than "Action Managers"